### PR TITLE
Added personalQr as a customer target

### PR DESCRIPTION
**Pull request by khenrichsen**

### CODE REVIEW

**Review by tht13 — APPROVED** [review]

File: docs/epayment.v1.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 openapi: 3.0.0
 servers:
-  - url: https://api.vipps.no
-    description: Production environment (uses the production API keys, the official app and live data)
-  - url: https://apitest.vipps.no
-    description: Test environment (uses the test API keys, the test app and test data)
+  - url: 'https://api.vipps.no'
```
Comment (madfr-at-vipps) [inline docs/epayment.v1.yml:3]: I'm super confused as to when you use single, double or no quotes for strings in the yaml file here, I'd love to understand though! :)

File: docs/epayment.v1.yml [hunk 2]
```diff
@@ -244,8 +243,12 @@ components:
             See: https://developer.vippsmobilepay.com/docs/vipps-developers/common-topics/vipps-landing-page/
           minLength: 10
           maxLength: 15
-          example: "4791234567"
+          example: '4791234567'
           pattern: '^\d{10,15}$'
+        personalQr:
+          type: string
+          description: 'The personal QR from the app, used in merchant scan scenarios. Pass on all information from the QR, do not format it in any way.'
```
Comment (madfr-at-vipps) [inline docs/epayment.v1.yml:250]: A little bit clunky to read this one.
What about something a little shorter?

`Personal QR code for merchant scan scenarios.`

or more reader friendly explaining the use case a bit more

` 
Personal QR code, allows user to scan the QR code at a point of sale with the app
and ...
`

I think given the context, that the QR code targets the app is a given,
and I'm confused as to why I'd want to format the information, so maybe
be more explicit or just remove that part and let people know when they 
should format stuff in a particular way.

File: docs/epayment.v1.yml (hunk 1 above)
Comment (khenrichsen) [reply docs/epayment.v1.yml:3]: autoformatting by stoplight.I can tidy it up!

**Review by madfr-at-vipps — APPROVED** [review]

File: docs/epayment.v1.yml (hunk 2 above)
Comment (khenrichsen) [reply docs/epayment.v1.yml:250]: Hm, i think ` Personal QR code, allows user to scan the QR code at a point of sale with the app and ` makes it more confusing. The personal QR is the qr from the vipps app, so this is used in contexts where the user show their own QR to the merchant. In the rest of the docs this is called merchant scan :) 

I had a shorter sentence first, but i followed the convention on the other descriptions which are also pretty long. Input @cloveras :)?

Comment (cloveras) [inline docs/epayment.v1.yml:250]: Attempt at making it clear. We should probably have a `oneOf` in the API spec for https://developer.vippsmobilepay.com/api/epayment/#tag/CreatePayments/operation/createPayment with both `phoneNumber` and something like `personalQrCode`, so we can validate both variants properly.

```suggestion
          description: 'The user's personal QR code in the app. This QR code can be scanned by merchants, and used in the `POST:/epayment/v1/payments` request to initiate a payment. **Important:** The content of the QR code can change at any time, without warning, so it is important that the _entire content of the QR code_ is specified as the `phoneNumber` when initiating payments. Do not attempt to extract the phone number from the QR code contents, as the content may change and no longer contain a phone number.'
```

Comment (cloveras) [reply docs/epayment.v1.yml:250]: Suggestion added! ✨

Comment (khenrichsen) [reply docs/epayment.v1.yml:250]: should this be in the description instead?
We shouldnt mention that the phonenumber is part of the QR at all.. I can try to rephrase it

Comment (cloveras) [reply docs/epayment.v1.yml:250]: I'm DM-ing @khenrichsen, and he will clean this up. 🧹

**Pull request merged**